zerver/webhooks/zapier/doc.html

```diff
@@ -40,3 +40,33 @@
 <p>You're done! Example message:</p>
 
 <img class="screenshot" src="/static/images/integrations/zapier/005.png"/>
+
+<p>
+    <b>Weather Update with Zapier!</b><br />
+</p>
+<p>
+    Go to the following link <code>https://zapier.com/app/explore</code> after you sign into your zapier account<br />
+    Select Send Weather info to your app via webhooks.
+
+    <img class="screenshot" src="/static/images/integrations/zapier/006.png" /><br /><br />
+    Select trigger app as <b>weather by zapier</b><br /><br />
```

Comment:
We wouldn't use "as" here and we should capitalize and punctuate appropriately:
~~~
Choose the trigger app "<b>Weather by Zapier</b>."
~~~